languageserver/src/main/scala/langserver/messages/Commands.scala

```diff
@@ -149,21 +149,20 @@ object ShutdownResult {
     OWrites[ShutdownResult](s => Json.obj()))
 }
 
+/**
+  * The show message request is sent from a server to a client to ask the client to display a
+  * particular message in the user interface. In addition to the show message notification the
+  * request allows to pass actions and to wait for an answer from the client.
+  *
+  * @param `type` The message type. @see [[MessageType]]
+  * @param message The actual message
+  * @param actions The message action items to present.
+  */
 case class ShowMessageRequestParams(
-  /**
-   * The message type. @see MessageType
-   */
-  tpe: Long,
-
-  /**
-   * The actual message
-   */
+  `type`: MessageType,
```

Comment:
I think this fixes a bug for which all messages where displayed as Info

Reply:
It's strange, but I don't get any of those messages now.. 🤔 how do I break it to get one of those warnings? 😄

Reply:
Editing a file, and try jumping to a definition without compiling it. It should prompt a message.

Reply:
I guess, I can't do it now because go-to-definition is broken for me.. Thanks anyway.